ynr/apps/people/managers.py

```diff
@@ -18,26 +18,6 @@ def create_from_file(self, image_filename, ideal_relative_name, defaults):
             storage_filename = storage.save(desired_storage_path, f)
         return self.create(image=storage_filename, **defaults)
 
-    def update_or_create_from_file(
```

Comment:
@symroe this method was not used in the code anywhere so we decided to remove it along with the `test_import.py` file which was testing it. It seemed to us like this method may have only been used in an old/removed functionality e.g. possibly an initial one off import of images?